Gemfile

```diff
@@ -139,6 +139,7 @@ gem "faraday"
 gem "private_address_check"
 
 group :production, :staging do
+  gem 'newrelic_rpm'
```

Comment:
Would it hurt to have it in all environments?

I find it increases the chance to catch compatibility bugs early.

Reply:
It don't think it would. I fixed up the configuration so we don't run the agent by default in Dev. And I realised I forgot to actually test if the configuration was working, which I have now done.